admin_guide/upgrading_citus.rst

```diff
@@ -39,7 +39,7 @@ Major and Minor Version Upgrades
 
 Major and minor version upgrades follow the same steps, but be careful: major upgrades can make backward-incompatible changes in the Citus API. It is best to review the Citus `changelog <https://github.com/citusdata/citus/blob/master/CHANGELOG.md>`_ before a major upgrade and look for any changes which may cause problems for your application.
 
-Each major and minor version of Citus is published as a package with a separate name. Installing a newer package will automatically remove the older version. Here is how to upgrade from 6.2 to 7.5 for instance:
+Each major and minor version of Citus is published as a package with a separate name. Installing a newer package will automatically remove the older version. Here is how to upgrade from 7.5 to 8.0 for instance:
```

Comment:
This needs to change. I think here we need to clarify that 7.5 won't work with PG11.

Reply:
We also need to replace 11 with 10 everywhere here. Any update from 7.5 to 8.0 will need to be first on PG10.

Reply:
I left pg 11 in the "patch update" section where it's assuming an upgrade within citus 8, but switched to 10 for anything involving 7.5